02.calendar/calendar.rb

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+require 'optparse'
+require 'date'
+
+params = ARGV.getopts("", "y:#{Date.today.year}", "m:#{Date.today.month}")
+
+year = params.values[0].to_i
+month = params.values[1].to_i
```

Comment:
```suggestion
year, month = params.values.map(&:to_i)
```

やや高等テクニックですが、こんなふうに書くこともできそうです。